src/components/QuranReader/ReadingView/Page.tsx

```diff
@@ -30,7 +30,7 @@ const Page = ({ verses, page, quranReaderStyles }: PageProps) => {
   );
   const isWordByWordLayout = showWordByWordTranslation || showWordByWordTransliteration;
   const isBigTextLayout =
-    isWordByWordLayout || quranTextFontScale > 3 || quranFont === QuranFont.Tajweed;
```

Comment:
3 was not chosen because it's the middle but rather it's the maximum font-scale we can apply using `flex` and above that the words will over-flow out of the QuranReader